packages/domains-table/src/domains-table/domains-table-ssl-cell.tsx

```diff
@@ -0,0 +1,62 @@
+import { DomainData } from '@automattic/data-stores';
+import { Icon } from '@wordpress/components';
+import { lock } from '@wordpress/icons';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+
+interface DomainsTableSSLCellProps {
+	domainManagementLink: string;
+	sslStatus: DomainData[ 'ssl_status' ];
+}
+
+export default function DomainsTableSSLCell( {
+	domainManagementLink,
+	sslStatus,
+}: DomainsTableSSLCellProps ) {
+	const translate = useTranslate();
+
+	const isPendingSSL = sslStatus === 'pending' || sslStatus === 'newly_registered';
```

Comment:
I realize now that if the end goal was to continue displaying `Pending` when the `ssl_status` is `newly_registered`, then we might have been able to land this earlier 🤷‍♂️ Anyway, aligning with the behavior in #93138 is the main goal, so this makes sense.